x-pack/plugins/actions/server/create_system_actions.ts

```diff
@@ -14,7 +14,7 @@ export const createSystemConnectors = (actionTypes: ActionType[]): InMemoryConne
   const systemConnectors: InMemoryConnector[] = systemActionTypes.map((systemActionType) => ({
     id: `system-connector-${systemActionType.id}`,
     actionTypeId: systemActionType.id,
-    name: `System action: ${systemActionType.id}`,
+    name: systemActionType.name,
```

Comment:
The name of a system connector will be the same name as the action type name. This change affected some tests.